recipes/boost/all/conanfile.py

```diff
@@ -76,6 +76,7 @@ class BoostConan(ConanFile):
         "system_no_deprecated": [True, False],
         "asio_no_deprecated": [True, False],
         "filesystem_no_deprecated": [True, False],
+        "filesystem_version": ["default", "3", "4"],
```

Comment:
Maybe `None` since there is no automatically default value?
```suggestion
        "filesystem_version": [None, "3", "4"],
```